gr-blocks/lib/selector_impl.cc

```diff
@@ -55,7 +55,10 @@ selector_impl::selector_impl(size_t itemsize,
     message_port_register_in(pmt::mp("en"));
     set_msg_handler(pmt::mp("en"), [this](pmt::pmt_t msg) { this->handle_enable(msg); });
 
-    // TODO: add message ports for input_index and output_index
+    message_port_register_in(pmt::mp("iindex"));
+    set_msg_handler(pmt::mp("iindex"), boost::bind(&selector_impl::handleMsgInputIndex, this, _1) );
+    message_port_register_in(pmt::mp("oindex"));
+    set_msg_handler(pmt::mp("oindex"), boost::bind(&selector_impl::handleMsgOutputIndex, this, _1) );
```

Comment:
Can these please be made boost-free as above ("en").

Reply:
Sure thing.  Standby I'll push an update for that and the header change.